nemo_curator/datasets/doc_dataset.py

```diff
@@ -155,7 +155,6 @@ def from_pandas(
                 npartitions=npartitions,
                 chunksize=chunksize,
                 sort=sort,
-                name=name,
```

Comment:
My understanding is that the `name` argument doesn't work for 24.08 (the corresponding dask-expr version has a small bug). Although `name` **is** supported for later dask-expr versions, I'm not sure how important it is for curator to support this argument? (I don't see it used internally anywhere, but could be wrong)